spec/models/section_spec.rb

```diff
@@ -110,6 +110,21 @@ def key_classes_for(hash)
     allow(SlugGenerator).to receive(:new).with(prefix: manual_slug).and_return(slug_generator)
   end
 
+  describe '#update_slug!' do
+    it 'updates the slug of the section' do
+      allow(SlugGenerator).to receive(:new).and_call_original
```

Comment:
Good spot. I've updated the commit note to explain why I'm using it:

> Note the use of `and_call_original` to restore the original behaviour of
> `SlugGenerator.new`. Without this, the stubbing of `SlugGenerator.new`
> in the main `before` block was complaining because it was expecting to
> be initialized with the slug that's set on the `manual` double for the
> rest of the tests.